src/getHumanAge.test.js

```diff
@@ -7,4 +7,44 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return [0, 0] for age 0 for both cat and dog', () => {
+    expect(getHumanAge(0, 0))
+      .toEqual([0, 0]);
+  });
+
+  test('should return [0, 0] for age 14 for both cat and dog', () => {
+    expect(getHumanAge(14, 14))
+      .toEqual([0, 0]);
+  });
+
+  test('should return [1, 1] for age 15 for both cat and dog', () => {
+    expect(getHumanAge(15, 15))
+      .toEqual([1, 1]);
+  });
+
+  test('should return [1, 1] for age 23 for both cat and dog', () => {
+    expect(getHumanAge(23, 23))
+      .toEqual([1, 1]);
```

Comment:
The test case description and expected result for age 23 returning [1, 1] might not be correct depending on the logic of `getHumanAge`. Verify if age 23 should indeed map to [1, 1] in human years.